src/BizHawk.Emulation.Common/Base Implementations/MemoryDomain.cs

```diff
@@ -119,13 +141,12 @@ public virtual void BulkPeekUshort(Range<long> addresses, bool bigEndian, ushort
 			var start = addresses.Start;
 			var end = addresses.EndInclusive + 1;
 
-			if ((start & 1) != 0 || (end & 1) != 0)
-				throw new InvalidOperationException("The API contract doesn't define what to do for unaligned reads and writes!");
+			if (start % 2 is not 0 || end % 2 is not 0) throw new InvalidOperationException(ERR_MSG_NOT_ALIGNED);

```

Comment:
Why are you changing this? `(start & 1) != 0` is easier to understand, and far nicer than `start % 2 is not 0`.

Reply:
Address alignment is inherently tied to bits, which makes the bitwise operator more appropriate.
I also much prefer the standard equality operator `==` over `is` and especially prefer `!=` over `is not`. (`is` would make sense with pattern matching, maybe even checking for `null`, but not for plain equality checks. And having whitespace inside the `is not` "operator" is just terrible imo.)
But even disregarding all that, this is a style change that isn't relevant to this PR.